app/controllers/events/view/tickets/access-codes/list.js

```diff
@@ -0,0 +1,33 @@
+import Ember from 'ember';
+
+const { Controller } = Ember;
+
+export default Controller.extend({
+  columns: [
+    {
+      propertyName : 'code',
+      title        : 'Access Code'
+    },
+    {
+      propertyName   : 'accessUrl',
```

Comment:
@sumedh123 Oh, I didn't know that, why is it so, I thought they are in not in camelcase because they are passed into the query as it is ?